<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KMusicListItemClass</class>
 <widget class="QWidget" name="KMusicListItemClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>93</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KMusicListItem</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>478</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="m_musicPic">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel#m_musicPic{
	border-image: url(:/images/music.png); 
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>18</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="m_title">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>35</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel#m_title{
	font-family: &quot;Microsoft YaHei&quot;;
    font-size: 17px;
    color: #BDC8E2;
}</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>17</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="m_singerAndAlbum">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>35</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel#m_singerAndAlbum{
	font-family: &quot;Microsoft YaHei&quot;;
    font-size: 17px;
    color: #BDC8E2;
}</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>18</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="m_duration">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>35</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel#m_duration{
	font-family: &quot;Microsoft YaHei&quot;;
    font-size: 17px;
    color: #BDC8E2;
}</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
